security/management/__init__.py

```diff
@@ -27,7 +27,8 @@ def execute_from_command_line_with_stdout(argv=None, stdout=None, stderr=None):
         from security.utils import CommandLogger
 
         # some arguments must be processed before django setup
-        parser = CommandParser(None, usage="%(prog)s subcommand [options] [args]", add_help=False)
+        parser_args = (None,) if django.VERSION < (2, 1) else tuple()
```

Comment:
Constructor signature changed between versions and 1.11 is incompatible with 2.1+.

Reply:
Maybe use function `get_main_version`? It'd in `django.utils.version`. I have bad experience when comparing versions directly like that, I was fixing chamber, because it was failing on bleeding edge django versions, since its version contains commit hashes, words etc.

Check out my PR in chamber:
https://github.com/druids/django-chamber/pull/90/files